source/package-json.d.ts

```diff
@@ -307,6 +307,7 @@ export type PackageJson = {
 	/**
 	The licenses for the package.
 	*/
+	// eslint-disable-next-line @typescript-eslint/array-type
```

Comment:
These are not needed. I think you have an outdated `node_modules` folder or outdated cache.

https://travis-ci.com/sindresorhus/type-fest/jobs/194522350

Reply:
Wasn't aware of it, thanks!